src/data/api.js

```diff
@@ -4,30 +4,20 @@
  * @param {String} moduleId Identifier of the module
  * @param {Object} customData THe data to send
  */
-function updateData(moduleId, customData) {
-  // PUT <iotinaot>/api/data {<custom_data>}, headers:{"Xiot-forward-to": model.__ip}
-  // fetch('http://www.iotinator.com/debug.htmlapi/data', {
-  //   credentials: 'include',
-  //   headers: {},
-  //   referrer: 'http://www.iotinator.com/debug.html',
-  //   referrerPolicy: 'no-referrer-when-downgrade',
-  //   body: '{"level":"46","id":"M5c_cf_7f_24_7a_ef"}',
-  //   method: 'PUT',
-  //   mode: 'cors'
-  // })
-
+function updateData(moduleIp, customData) {
   return new Promise((resolve /*, reject*/) => {
     // Simulate long-running renaming
-    setTimeout(() => {
-      // eslint-disable-next-line
-      console.log(
-        `Update custom data of '${moduleId}' with '${JSON.stringify(
-          customData
-        )}'`
-      )
-      resolve()
-    }, 2000)
+    fetch(document.location.origin + '/api/data', {
+      credentials: 'include',
+      headers: { 'Xiot-forward-to': moduleIp },
+      body: JSON.stringify(customData),
+      method: 'POST'
+    })
```

Comment:
`resolve` should be called in the `.then` method of the `fetch` call. `reject` should be uncommented and called in the `.catch` method.
```js
fetch(...)
.then(postData => {
  resolve(postData)
})
.catch(e=>reject(e))
```

Reply:
Done

Reply:
Done